src/main/java/de/unistuttgart/crosswordbackend/data/OverworldResultDTO.java

```diff
@@ -13,7 +13,7 @@
 @FieldDefaults(level = AccessLevel.PRIVATE)
 public class OverworldResultDTO {
 
-    final String game = "CROSSWORDPUZZLE";
+    final static String game = "CROSSWORDPUZZLE";
```

Comment:
Nope, not a good idea.
Mark this problem as "cannot be resolved" in SonarQube.
Otherwise, you break sending the results to the overworld backend.